Doc/library/csv.rst

```diff
@@ -416,7 +417,7 @@ Reader objects have the following public attributes:
 
 .. attribute:: csvreader.line_num
 
-   The number of lines read from the source iterator. This is not the same as the
+   The number of lines read from the source iterable. This is not the same as the
```

Comment:
Why change this? If there is only one iterator involved, the original would be more precise.